core/test/gather/gatherers/full-page-screenshot-test.js

```diff
@@ -4,8 +4,14 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+import * as puppeteer from 'puppeteer-core';
+import {getChromePath} from 'chrome-launcher';
+
+import * as LH from '../../../../types/lh.js';
```

Comment:
This shouldn't be necessary for a test file

Reply:
Ah this test is excluded in the `tsconfig.json`, so the normal `LH` global isn't available. I guess this is fine for now.